src/assets/components/tiles/Tile.js

```diff
@@ -0,0 +1,14 @@
+
+
+function Tile({image, imgDescription, title, children}) {
+    return (
+
+        <section>
+            <img src={image} alt={imgDescription}/>
```

Comment:
Je zou hier nog een voorwaarde toe kunnen voegen zodat hij de image-tag alleen genereert wanneer er ook een image is. In je tekst-tiles staat nu in de html een lege img-tag. 

```suggestion
            {img && <img src={img} alt={imgDescription}/>}
```
![Scherm­afbeelding 2023-05-17 om 09 28 25](https://github.com/RaymondvdWal/handbags/assets/23195288/b46548af-6ed6-49e4-a990-ab2b51238152)